.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_size = 2
+indent_style = space
+trim_trailing_whitespace = true
```

Comment:
[EditorConfig](http://editorconfig.org) is super helpful and there are [plugins](http://editorconfig.org/#download) for a bunch of editors.